config/secrets.yml

```diff
@@ -0,0 +1,22 @@
+# Be sure to restart your server when you modify this file.
+
+# Your secret key is used for verifying the integrity of signed cookies.
+# If you change this key, all old signed cookies will become invalid!
+
+# Make sure the secret is at least 30 characters and all random,
+# no regular words or you'll be exposed to dictionary attacks.
+# You can use `rake secret` to generate a secure secret key.
+
+# Make sure the secrets in this file are kept private
+# if you're sharing your code publicly.
+
+development:
+  secret_key_base: 96b1017307fba0e409032eb3a5ddf3634ef674b1bba34350f5fcc326a41d62c4f2057d67b2e3c4f899aa1d89a5f2f468e96d96862422c7ed3e27e09fc87aecbf
+
+test:
+  secret_key_base: 34319add6c897e98fe6adc6c70267512f388defd3a1a67a99e7647d8580b57b4b07417f96945d2163ff3ff4f7dfd83059d4f0078b6cf432cfde5f2b54e08b4ed
+
+# Do not keep production secrets in the repository,
+# instead read values from the environment.
+production:
+  secret_key_base: 878c2e50326751fd190bb9f83ae6e5af7de2f734d94e14549fea67923ad23ff0494f027565760672201adf42b1df8a101be946522616cbebb5329607a5fb30c1
```

Comment:
This project would be opensource and so vulnerable to expose production secret key.
 
Check http://guides.rubyonrails.org/upgrading_ruby_on_rails.html#config-secrets-yml